olp-cpp-sdk-core/include/olp/core/client/ErrorCode.h

```diff
@@ -21,66 +21,72 @@
 
 namespace olp {
 namespace client {
+
+/**
+ * @brief The ErrorCode enum represents all possible errors that might happen
+ * during a user request.
+ */
 enum class ErrorCode {
   /**
-   * Unknown error, see error message for details
+   * Unknown error, see error message for details.
    */
   Unknown = 0,
 
   /**
-   * Request was cancelled (usually by the user)
+   * Request was cancelled (usually by the user).
    */
   Cancelled,
 
   /**
-   * Request passed invalid aruguments
+   * Request passed invalid arguments.
    */
   InvalidArgument,
 
   /**
-   * Request exceeded timeout limit
+   * Request exceeded timeout limit.
    */
   RequestTimeout,
 
   /**
-   * Internal server failure
+   * Internal server failure.
    */
   InternalFailure,
 
   /**
-   * Request service is unavailable
+   * Request service is unavailable.
    */
   ServiceUnavailable,
 
   /**
-   * Access denied to service due to insufficient credentials
+   * Access denied to service due to insufficient credentials.
    */
   AccessDenied,
 
   /**
-   * Request malformed
+   * Request malformed.
```

Comment:
What do you mean by "malformed"?

Reply:
URL malformed or some data parameters are not formed correctly. It is the equivalent of HTTP 400 Status:

```
400 Bad Request
The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, size too large, invalid request message framing, or deceptive request routing).
```